docs(api/conventions): mention `actionHeaders` in `headers`' arguments (#3167)
remix-run/remix: docs/api/conventions.md

Each route can define its own HTTP headers. One of the common headers is the `Cache-Control` header that indicates to browser and CDN caches where and for how long a page is able to be cached.

```tsx

> export function headers({ loaderHeaders, actionHeaders, parentHeaders }) {

Reviewer: ```suggestion
export function headers({ actionHeaders, loaderHeaders, parentHeaders }) {
```

  return {
    "X-Stretchy-Pants": "its for fun",
    "Cache-Control": "max-age=300, s-maxage=3600",
  };
}
```

> Usually your data is a better indicator of your cache duration than your route module (data tends to be more dynamic than markup), so the loader's and action's headers are passed in to `headers()` too:

Reviewer: ```suggestion
Usually your data is a better indicator of your cache duration than your route module (data tends to be more dynamic than markup), so the `action`'s & `loader`'s headers are passed in to `headers()` too:
```

}
```

> Note: `loaderHeaders` and `actionHeaders` are an instance of the [Web Fetch API][headers] `Headers` class.

Reviewer: ```suggestion
Note: `actionHeaders` & `loaderHeaders` are an instance of the [Web Fetch API][headers] `Headers` class.
```
